CHANGELOG.md

```diff
@@ -1,3 +1,133 @@
+ * fix(localpv-hostpath): pv deletion caused panic when node not found ([#1662](https://github.com/openebs/maya/pull/1662),
+ [@kmova](https://github.com/kmova))
+ * feat(build): enable arm64 auto build ([#1650](https://github.com/openebs/maya/pull/1650),
+ [@akhilerm](https://github.com/akhilerm))
+ * chore(build): trigger downstream repo release ([#1657](https://github.com/openebs/maya/pull/1657),
+ [@kmova](https://github.com/kmova))
+ * feat(upgrade): enable bulk upgrade for volumes and spc ([#1655](https://github.com/openebs/maya/pull/1655),
+ [@shubham14bajpai](https://github.com/shubham14bajpai))
+ * feat(localpv-device): allow local pv device on select devices ([#1648](https://github.com/openebs/maya/pull/1648),
+ [@kmova](https://github.com/kmova))
+ * feat(build): adding pc64le build for local provisioner ([#1632](https://github.com/openebs/maya/pull/1632),
+ [@Pensu](https://github.com/Pensu))
+ * fix(BDD): fix SPC reconciliation BDD by adding extra filter ([#1654](https://github.com/openebs/maya/pull/1654),
+ [@mittachaitu](https://github.com/mittachaitu))
+ * feat(upgrade): split jiva replicas and migrate jiva resources to openebs ([#1646](https://github.com/openebs/maya/pull/1646),
+ [@shubham14bajpai](https://github.com/shubham14bajpai))
+ * feat(cstorBackup, delete): support for snapshot deletion ([#1644](https://github.com/openebs/maya/pull/1644),
+ [@mynktl](https://github.com/mynktl))
+ * fix(bdd): fix BDD to avoid failures during creation of StorageClass ([#1652](https://github.com/openebs/maya/pull/1652),
+ [@mittachaitu](https://github.com/mittachaitu))
+ * fix(jiva): add namespace for stateful set target affinity([#1651](https://github.com/openebs/maya/pull/1651),
+ [@shubham14bajpai](https://github.com/shubham14bajpai))
+ * fix(bdd): update the namespace for verifying jiva pods ([#1649](https://github.com/openebs/maya/pull/1649),
+ [@shubham14bajpai](https://github.com/shubham14bajpai))
+ * feat(api): add label selector in BDC spec ([#1647](https://github.com/openebs/maya/pull/1647),
+ [@akhilerm](https://github.com/akhilerm))
+ * feat(BDD): add positive test cases for verifying waitforfirstconsumer With CStor Volume Provisioning ([#1643](https://github.com/openebs/maya/pull/1643),
+ [@mittachaitu](https://github.com/mittachaitu))
+ * fix(jiva): add namespace to podAffinity for target([#1645](https://github.com/openebs/maya/pull/1645),
+ [@shubham14bajpai](https://github.com/shubham14bajpai))
+ * chore(BDD): checks to verify pool protectionfinalizer ([#1640](https://github.com/openebs/maya/pull/1640),
+ [@mittachaitu](https://github.com/mittachaitu))
+ *  feat(estimate_rebuilds): add pending snapshots on CVR by talking to peer replicas ([#1641](https://github.com/openebs/maya/pull/1641),
+ [@mittachaitu](https://github.com/mittachaitu))
+ * feat(apis): add new fields into BlockDevice([#1642](https://github.com/openebs/maya/pull/1642),
+ [@akhilerm](https://github.com/akhilerm))
```

Comment:
This need not be added in changelog. It is related to migrating fields from NDM to here.